test/servinge2e/helpers.go

```diff
@@ -44,3 +53,128 @@ func withServiceReadyOrFail(ctx *test.Context, service *servingv1.Service) *serv
 
 	return service
 }
+
+type telemetryStat struct {
+	services       float64
+	routes         float64
+	configurations float64
+	revisions      float64
+}
+
+func getMetricsEndpointPath() string {
+	portAndPath := strconv.Itoa(8383) + "/metrics"
+	metricsPath := "http://knative-openshift-metrics.openshift-serverless:" + portAndPath
+	return metricsPath
+}
```

Comment:
This won't work as the hostname is hardcoded. You'll have to fetch it from the route after it's applied and accepted. The port will also be 80.

Reply:
Yeah I need the domain.